doc/source/whatsnew/v0.25.0.rst

```diff
@@ -247,7 +249,7 @@ Datetimelike
 Timedelta
 ^^^^^^^^^
 
--
+- Bug in :func:`TimedeltaIndex.intersection` where for non-monotonic indicies in some cases an empty Index was returned when in fact an intersection existed.
```

Comment:
Is there an issue number for this?  Otherwise, reference the PR as the "issue".

Reply:
This came up during the implementation of the sort keyword. I have referenced the PR as issue.